packages/contracts-bsc/contracts/BscTokenController.sol

```diff
@@ -533,10 +534,10 @@ contract BscTokenController {
     }
 
     /**
-     * @dev Claim ownership of an arbitrary HasOwner contract
-
+     * @dev Claim ownership of an arbitrary IHasOwner contract
+     */
     function issueClaimOwnership(address _other) public onlyOwner {
-        HasOwner other = HasOwner(_other);
+        IHasOwner other = IHasOwner(_other);
         other.claimOwnership();
     }
 
```

Comment:
If you're bringing back `claimTrueCurrencyImplOwnership()`, then, for symmetry, I would also bring back the commented out function below (and move it to an `impl functions` section, see above comment). Something like this? Should be done in the same PR.

```
function transferTrueCurrencyImplOwnership(address _newOwner) external onlyOwner {
    token.transferOwnership(_newOwner);
}
```

Reply:
There is a pair of functions `issueClaimOwnership` and `transferChild` to manage the ownership of `token`(or some other token maybe, I have no idea about the original design). I think it would be better to keep the name. But I will comment out transferChild in case we need it.

Reply:
I am also uncertain about the original intent -- I wasn't at Archblock when this design decision was made.

However, what I can say is that functions should have descriptive names that are consistent with the local style. My best judgment is that `issueClaimOwnership()` + `transferChild()` must have been remnants of previous naming refactors, but were skipped because they were commented out.

However, as of _now_, these two functions are:
- no longer descriptive (`child` does not appear to be used anywhere else in this contract, so it's not obvious from the name what it refers to -- we can _infer_ that it refers to `token`, but, if so, then we should definitely _not_ keep the name `child`),
- nor consistent with local style (no other function is named `issueX()` -- there are still functions named `requestX()` for `token.X()` and `X()` for `token.X()`, but none named `issueX()`).

If we want to adhere to descriptiveness and consistency, then we should name these two functions something like `claimTrueCurrencyImplOwnership()` + `transferTrueCurrencyImplOwnership()`, and, as I mention in [this comment](https://github.com/trusttoken/contracts-pre22/pull/1245#discussion_r1145470568), move these functions to a new `impl functions` section below `proxy functions` so that they correspond with the existing `claimTrueCurrencyProxyOwnership()` and `transferTrueCurrencyProxyOwnership()` functions.